src/django/pfb_analysis/models.py

```diff
@@ -272,7 +276,7 @@ def _set_geom_from_boundary_file(self, overwrite=False):
                 shutil.rmtree(tmpdir, ignore_errors=True)
 
     class Meta:
-        unique_together = ('name', 'state_abbrev', 'organization',)
+        unique_together = ('name', 'country', 'state_abbrev', 'organization',)
```

Comment:
I noticed that the `boundary_file` path on the new neighborhoods outside the US looked pretty simple, and I confirmed that if I make two neighborhoods with the same Label in two different countries, it works fine but the boundary file for the second clobbers the first one on S3.

Turns out these same `unique_together` fields are used in [get_neighborhood_file_upload_path](https://github.com/azavea/pfb-network-connectivity/blob/0.7.0/src/django/pfb_analysis/models.py#L43-L48) to generate that path.  So we should add country into the mix.  (And maybe add comments in both places to alert future-us to this nuance.)

Reply:
I assume we'll need to maintain backwards compatibility with previously uploaded files as well.

Reply:
I'm not sure we do, actually.  I think that's run only when the field is saved and the value is stored (in the database, the `boundary_file` field is just a `VARCHAR[1024]`) for use in lookups.

But the `obj.state_abbrev or obj.country.alpha3` way works fine, and will keep the bucket cleaner since changing the path for existing neighborhoods would mean that updating their geometry would leave a stale copy at the old path.

I do think a comment here noting that changing it might affect upload path generation and a note in the `get_neighborhood_file_upload_path` docstring that any terms it includes need to be either unique in themselves (e.g. job_id) or included in the `unique_together` would be helpful for posterity.

Reply:
Comments updated again.